### PR TITLE
Frozen Enemy Runways in Norfair West

### PR DESCRIPTION
**Pull request by kjbranch**
Here is Insomnia's video for the Geruta freeze: https://discord.com/channels/1053421401354285129/1053436236628496454/1156796076339245066

It's possible but terrible to freeze the Geruta to extend the left runway, but I'm leaving that out.

PS: There were no more runways needed in Crocomire's section, so that's done too.

### CODE REVIEW

File: region/norfair/west/Crocomire Escape.json [hunk 1]
```diff
@@ -139,8 +151,44 @@
...
+      "name": "Croc Escape Frozen Geruta Shinecharge (Shinespark Out Right)",
+      "reusableRoomwideNotable": "Croc Escape Frozen Geruta Shinecharge",
+      "notable": true,
+      "entranceCondition": {
+        "comeInNormally": {}
+      },
+      "requires": [
+        "h_canTrickyFrozenEnemyRunway",
+        "canCarefulJump",
+        {"enemyDamage": {
```
Comment (osse101) [inline region/norfair/west/Crocomire Escape.json:167]: This hit is avoidable at this level with another freeze.
Morph sometimes works but isn't perfect.

File: region/norfair/west/Crocomire Escape.json [hunk 2]
```diff
@@ -139,8 +151,44 @@
...
+          "enemy": "Geruta",
+          "type": "contact",
+          "hits": 1
+        }},
+        {"canShineCharge": {
+          "usedTiles": 17,
+          "openEnd": 1
+        }},
+        "canShinechargeMovementComplex",
```
Comment (osse101) [inline region/norfair/west/Crocomire Escape.json:176]:
```suggestion
        "canShinechargeMovementTricky",
```

File: region/norfair/west/Crocomire Escape.json [hunk 3]
```diff
@@ -314,8 +362,46 @@
...
+      "link": [2, 3],
+      "name": "Croc Escape Frozen Geruta Shinecharge (Shinespark Left)",
+      "reusableRoomwideNotable": "Croc Escape Frozen Geruta Shinecharge",
+      "notable": true,
+      "entranceCondition": {
+        "comeInNormally": {}
+      },
+      "requires": [
+        "h_canTrickyFrozenEnemyRunway",
+        "canCarefulJump",
```
Comment (osse101) [inline region/norfair/west/Crocomire Escape.json:377]:
```suggestion
        "canCarefulJump",
        "canHorizontalShinespark",
```

File: region/norfair/west/Ice Beam Snake Room.json [hunk 4]
```diff
@@ -126,6 +126,26 @@
...
+        "leaveWithRunway": {
+          "length": 10,
+          "openEnd": 1
+        }
+      },
+      "note": [
+        "Wait 80 seconds for the Sova to get into position, then freeze it to bridge the gap and significantly extend the runway.",
+        "The other Sovas can be farmed or used as a backup."
+      ],
+      "devNote": "The heat frames can be reduced if starting from either of the other doors, but it will likely require heatProof either way."
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/norfair/west/Ice Beam Snake Room.json:147]: I'll reword this

**Pull request merged**